packages/hooks/use-teleport/index.ts

```diff
@@ -0,0 +1,61 @@
+import { h, ref, Teleport, onUnmounted } from 'vue'
+import { NOOP } from '@vue/shared'
+import isServer from '@element-plus/utils/isServer'
+import { createGlobalNode, removeGlobalNode } from '@element-plus/utils/global-nodes'
+
+import type { VNode, Ref } from 'vue'
+
+export default (contentRenderer: () => VNode, appendToBody: Ref<boolean>) => {
```

Comment:
Does appendToBody must be a ref? or Ref | boolean?

Reply:
I think this should be a ref since the flag was passed from the user we better keep tracking this.